File: src/main/java/com/selqet/momrecipebook/repository/IngredientRepository.java
package com.selqet.momrecipebook.repository;

import com.selqet.momrecipebook.entity.IngredientEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

public interface IngredientRepository extends JpaRepository<IngredientEntity, Long> {

}

File: src/main/kotlin/com/example/momrecipebook/datasource/IngredientRepository.kt
package com.example.momrecipebook.datasource

import com.example.momrecipebook.model.IngredientEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface IngredientRepository : CrudRepository<IngredientEntity, Long> {
    fun findByOrderByName(): List<IngredientEntity>
    fun deleteByNameEqualsIgnoreCase(name: String)
}